universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -131,6 +130,8 @@ private CsvExportConfig generateDefaultCsvConfig(ImmutableList<AnswerData> answe
     columnsBuilder.add(Column.builder().setHeader("ID").setColumnType(ColumnType.ID).build());
     columnsBuilder.add(
         Column.builder().setHeader("Submit time").setColumnType(ColumnType.SUBMIT_TIME).build());
+    columnsBuilder.add(
+        Column.builder().setHeader("Language").setColumnType(ColumnType.LANGUAGE).build());
```

Comment:
nit: maybe "Applicant Language" would make it clearer this is the applicant's preferred language? Either way is fine though!

Reply:
Good suggestion. done!